src/atlas/interpolation/method/sphericalvector/SphericalVector.h

```diff
@@ -0,0 +1,127 @@
+/*
+ * (C) Crown Copyright 2023 Met Office
+ *
+ * This software is licensed under the terms of the Apache Licence Version 2.0
+ * which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+#include "atlas/library/defines.h"
+
+#pragma once
+
+#include <complex>
+#include <memory>
+
+#if ATLAS_HAVE_EIGEN
+#include <Eigen/Sparse>
+#endif
+
+#include "atlas/functionspace/FunctionSpace.h"
+#include "atlas/interpolation/method/Method.h"
+#include "atlas/linalg/sparse.h"
+#include "eckit/config/Configuration.h"
```

Comment:
The correct include is "eckit/config/LocalConfiguration.h", and should precede the "atlas/"* includes

Reply:
Done. Also moved to the .cc along with constructor definition.